apps/forgejo/overlays/nishir-tailnet/kustomization.yaml

```
@@ -16,8 +19,14 @@ patches:
       name: forgejo
       version: v1
       kind: NetworkPolicy
-  - path: patch-svc.yaml
 secretGenerator:
   - name: forgejo-config
     files:
       - app.ini=forgejo/app.enc.ini
+patches:
```

YAMLlint (1.37.1)

[error] duplication of key "patches" in mapping

(key-duplicates)